src/pages/reporting.tsx

```diff
@@ -1,7 +1,128 @@
 import { NextPageWithLayout } from "./_app";
 
+import React, { PureComponent } from 'react';
+import { BarChart, Bar, Cell, XAxis, YAxis, CartesianGrid, Tooltip, Legend, ResponsiveContainer, PieChart, Pie } from 'recharts';
+import { Button } from "@/components/atoms";
+
+const data = [
+  {
+    name: 'Winter',
+    class1: 600,
+    class2: 400,
+    class3: 523,
+  },
+  {
+    name: 'Summer',
+    class1: 660,
+    class2: 500,
+    class3: 670,
+  },
+  {
+    name: 'Spring',
+    class1: 680,
+    class2: 700,
+    class3: 690,
+  },
+];
+
+const data01 = [
+  { name: 'Group A', value: 400 },
+  { name: 'Group B', value: 300 },
+  { name: 'Group C', value: 300 },
+  { name: 'Group D', value: 200 },
+  { name: 'Group E', value: 278 },
+  { name: 'Group F', value: 189 },
+];
+
+const colors: string[] = ["#FFBF00", "#DE3163", "#CCCCFF", "#6495ED", "#9FE2BF", "#40E0D0"];
+
 const ReportingPage: NextPageWithLayout = () => {
-  return <div>Reporting & Analysis</div>;
+  return (<div className="flex flex-row w-full py-24" style={{justifyContent: 'space-around', overflow: 'auto'}}>
```

Comment:
was prettier run here?

just needs linting otherwise looks good

Reply:
No I forgot to run linting lol, will do

Reply:
Updated